common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3328,6 +3328,12 @@
     ],
     "sqlState" : "42000"
   },
+  "NULL_DATA_SOURCE_OPTION" : {
+    "message" : [
+      "JDBC Option value <option> cannot have null value."
```

Comment:
let's make it general and not mention JDBC. Just say `Data source read/write option <option> cannot have null value.`

Reply:
Resolved.